sdk/examples/how_tos/advanced_transactions/advanced_transaction.rs

```diff
@@ -37,18 +38,14 @@ async fn main() -> Result<()> {
             .set_stronghold_password(std::env::var("STRONGHOLD_PASSWORD").unwrap())
             .await?;
 
-        // Create an ouput with amount 1_000_000 and a timelock of 1 hour
-        let in_an_hour = (std::time::SystemTime::now() + std::time::Duration::from_secs(3600))
-            .duration_since(std::time::UNIX_EPOCH)
-            .expect("clock went backwards")
-            .as_secs()
-            .try_into()
-            .unwrap();
+        // TODO better time-based UX ?
```

Comment:
Nit: reminder for this one too?

Reply:
Added in https://github.com/iotaledger/iota-sdk/issues/647